### PR TITLE
Add Default ErrorHandling

### PR DESCRIPTION
**Pull request by Pikkupomo**
When project doesn't define handlers, we inject our own handlers.

Refs SHOOP-1490

### CODE REVIEW

File: shoop/front/__init__.py [hunk 1]
```diff
@@ -25,6 +28,35 @@ class ShoopFrontAppConfig(AppConfig):
...
+        Install custom error handlers.
+
+        Error handlers to be added are for errors 400, 403, 404, and 500.
+
+        Error handlers will be injected only if:
+        * `settings.SHOOP_FRONT_INSTALL_ERROR_HANDLERS` is `True`
+        * `settings.ROOT_URLCONF` doesn't already contain the handler
+        """
+
+        if not getattr(django.conf.settings, "SHOOP_FRONT_INSTALL_ERROR_HANDLERS", True):
```
Comment (akx) [inline shoop/front/__init__.py]: All settings should be declared in the respective app's settings.py

**Comment by akx** [comment]
retest this please

**Comment by Pikkupomo** [comment]
retest this please

**Comment by akx** [comment]
I don't think the "Fix login after activating" commit should be here.

**Comment by akx** [comment]
retest this please

**Pull request merged**